engine/gamesys/src/dmsdk/gamesys/script.h

```diff
@@ -101,19 +101,42 @@ namespace dmScript
      *
      * @struct
      * @name dmScript::LuaHBuffer
+     * @member Union of
+     *     - m_BufferRes [type:void*]                       A buffer resource
+     *     - m_Buffer    [type:dmBuffer::HBuffer]           A buffer
      * @member m_Buffer [type:dmBuffer::HBuffer]            The buffer (or resource)
      * @member m_Owner  [type:dmScript::LuaBufferOwnership] What ownership the pointer has
+     *
+     * @examples
+     *
+     * See examples for dmScript::PushBuffer()
+     *
      */
     struct LuaHBuffer
     {
         union {
             dmBuffer::HBuffer   m_Buffer;
             void*               m_BufferRes;
         };
-        union {
+        union
+        {
             bool                m_UseLuaGC; // Deprecated
```

Comment:
Really want to remove the deprecated member, but we need to cleanup the extensions that use the old way of create the LuaHBuffer struct.